src/org/parosproxy/paros/model/SiteNode.java

```diff
@@ -230,6 +230,33 @@ public boolean isParentOf (String nodeName) {
     public String getNodeName() {
     	return this.nodeName;
     }
+    
+	/**
+	 * Returns the node's name optionally with or without the HTTP Method (verb)
+	 * prefixing the string.
+	 * 
+	 * @param withMethodPrefix
+	 *            a {@code boolean} specifying whether or not the returned
+	 *            string should include the HTTP Method as a prefix.
+	 * @return the name of the site node
+	 * @see #getNodeName()
+	 * @see #getCleanNodeName()
+	 * @see #getCleanNodeName(boolean)
+	 * @since TODO add version
+	 */
+    public String getNodeName(boolean withMethodPrefix) {

```

Comment:
I'd suggest dropping the boolean parameter and return always without the HTTP method (no need to duplicate the existing behaviour). Regarding the name, it could well be `getName()` or if we want to keep the "Node" `getNodeNameWithoutMethod()`.